src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/MediaPicker/ImageListQuery/ImageListQuery.jsx

```diff
@@ -14,13 +14,15 @@ const ImageListQueryCmp = ({
     selectedPath,
     initialSelection
 }) => {
-    const {data, error, loading} = useQuery(MediaPickerImages, {
+    const {data, error, loading, refetch} = useQuery(MediaPickerImages, {
         variables: {
             typeFilter: ['jmix:image'],
             path: selectedPath
         }
     });
 
+    window.refetchCallbacks = [refetch];
+
```

Comment:
Can you push into the array instead of assign in it if the array already exists?

Reply:
done